src/reporters/console/console-reporter.js

```diff
@@ -27,6 +27,12 @@ const tty = require('tty');
 type Row = Array<string>;
 type InquirerResponses<K, T> = {[key: K]: Array<T>};
 
+// fixes bold on windows
+// $FlowFixMe TERM is not a string?
+if (process.platform === 'win32' && !/^xterm/i.test(process.env.TERM)) {
```

Comment:
Does `flow` still complain if you do
```js
if (process.platform === 'win32' && process.env.TERM && !/^xterm/i.test(process.env.TERM)) {
```

And, how about using `startswith` instead of the regex?